changelog.md

```diff
@@ -1,3 +1,111 @@
+* throw on excess placeholder replacement values ([#9580](https://github.com/googlechrome/lighthouse/pull/9580))
+* remove translated messages when ICU arguments change ([#9598](https://github.com/googlechrome/lighthouse/pull/9598))
+* use better types for intl-messageformat ([#9570](https://github.com/googlechrome/lighthouse/pull/9570))
+* add code spans to uses-rel-preconnect description ([#9568](https://github.com/googlechrome/lighthouse/pull/9568))
+* fix bugs in locale-importing scripts ([#9621](https://github.com/googlechrome/lighthouse/pull/9621))
+* fix custom formatted ICU within plurals ([#9460](https://github.com/googlechrome/lighthouse/pull/9460))
+* capitalize robots.txt displayValue ([#9567](https://github.com/googlechrome/lighthouse/pull/9567))
+* add link to cldr parentLocales ([#9520](https://github.com/googlechrome/lighthouse/pull/9520))
+* move pwa audits to i18n system ([#9105](https://github.com/googlechrome/lighthouse/pull/9105))
+* use tsc to collect UIStrings ([#9487](https://github.com/googlechrome/lighthouse/pull/9487))
+* align type names with docs ([#9461](https://github.com/googlechrome/lighthouse/pull/9461))
+* expand placeholder support ([#9114](https://github.com/googlechrome/lighthouse/pull/9114))
+* roll: roll es-419, fr, id, zh-TW locales. correct fr ([#9454](https://github.com/googlechrome/lighthouse/pull/9454))
+* rewrite td-headers-attr titles ([#9587](https://github.com/googlechrome/lighthouse/pull/9587))
+
+## Report
+
+* accessibility: make dropdown match ARIA action menu button pattern ([#9433](https://github.com/googlechrome/lighthouse/pull/9433))
+* refactor: Move drop down logic into a separate class ([#9564](https://github.com/googlechrome/lighthouse/pull/9564))
+* topbar: fix overflowing url ([#9497](https://github.com/googlechrome/lighthouse/pull/9497))
+* viewer: add option for loading from the PSI API ([#9546](https://github.com/googlechrome/lighthouse/pull/9546))
+* don't use unnessary monospace ([#9626](https://github.com/googlechrome/lighthouse/pull/9626))
+* remove PWA badge flicker workaround for Chrome ([#9636](https://github.com/googlechrome/lighthouse/pull/9636))
+* add utm params to web.dev links ([#9555](https://github.com/googlechrome/lighthouse/pull/9555))
+
+## Tests
+
+* migrate travis to xenial for modern Chrome support ([#9654](https://github.com/googlechrome/lighthouse/pull/9654))
+* fixtures: use relative path in static server 404 page ([d0e5c58](https://github.com/googlechrome/lighthouse/commit/d0e5c58))
+* report: add axe-core validation of report output ([#9421](https://github.com/googlechrome/lighthouse/pull/9421))
+* smokehouse: index for static server. print address ([#9541](https://github.com/googlechrome/lighthouse/pull/9541))
+* smokehouse: use mime-types + proper encoding for response write ([#9542](https://github.com/googlechrome/lighthouse/pull/9542))
+
+## Misc
+
+* new import control text ([#9595](https://github.com/googlechrome/lighthouse/pull/9595))
```

Comment:
should move to `clients` and use the PR's `proto: update import hooks, obey line length`